root/relationship/linkattributetype/RelationshipAttributeTypeIndex.js

```diff
@@ -0,0 +1,197 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+import {upperFirst} from 'lodash';
+
+import Layout from '../../layout';
+import expand2react from '../../static/scripts/common/i18n/expand2react';
+import linkedEntities from '../../static/scripts/common/linkedEntities';
+import bracketed, {bracketedText}
+  from '../../static/scripts/common/utility/bracketed';
+import formatEntityTypeName
+  from '../../static/scripts/common/utility/formatEntityTypeName';
+import compareChildren from '../utility/compareChildren';
+
+type Props = {
+  +$c: CatalystContextT,
+  +attribute: LinkAttrTypeT,
+  +relationships: Array<LinkTypeT>,
+};
+
+const AttributeTree = ({attribute}: {attribute: LinkAttrTypeT}) => {
+  const childrenAttrs = attribute.children || [];
+  return (
+    <li style={{marginTop: '0.25em'}}>
+      <strong>
+        <a href={'/relationship-attribute/' + attribute.gid}>
+          {upperFirst(l_relationships(attribute.name))}
+        </a>
+      </strong>
+      {' '}
+      {attribute.description ? (
+        <>
+          {bracketed(expand2react(l_relationships(attribute.description)))}
+          {' '}
+        </>
+      ) : null}
+      {bracketedText(attribute.child_order.toString())}
+
+      {childrenAttrs.length ? (
+        <ul>
+          {childrenAttrs
+            .slice(0)
+            .sort((a, b) => compareChildren(a, b))
```

Comment:
Don't these already come sorted from the server? Seems like they should be if not. But anyway, you can pass the sort function directly here:
```suggestion
            .sort(compareChildren)
```

Reply:
They don't, currently, no. But did change to pass the function directly at least.